kolibri/core/assets/src/views/CoreBase/index.vue

```diff
@@ -314,7 +314,7 @@
       },
       notAuthorized() {
         // catch "not authorized" error, display AuthMessage
-        if (this.error && this.error.status && this.error.status.code == 403) {
```

Comment:
When going to an invalid resource ID on Learn, `this.error` isn't an object, but a string `"Error: Request failed with status code 404"`

Reply:
Seems that this was a result of Axios errors actually being Javascript Error objects, so it was just getting stringified rather than JSONified.